CMakeLists.txt

```diff
@@ -140,23 +151,26 @@ else()
             $<BUILD_INTERFACE:${fast_float_SOURCE_DIR}/include>
             $<INSTALL_INTERFACE:include>
             )
+    message(STATUS "Installed fast_float version ${PLSSVM_fast_float_VERSION}.")
 endif()
 
 ## try finding fmt
-find_package(fmt QUIET)
+find_package(fmt 8.0.0 QUIET)
```

Comment:
Have we checked if this is the minimum version, or is it any that PLSSVM currently runs with?